interactive-cherry-pick/README.md

```diff
@@ -0,0 +1,28 @@
+# Interactive Cherry-Pick Tool
+
+This is parallel to interactive rebasing in git, but for cherry-picks from a
+main branch to a stable release branch.
+
+
+## Installation
+
+sudo ln -s $(pwd)/interactive-cherry-pick.sh /usr/local/bin/interactive-cherry-pick
+
+
+## Usage
+
+```sh
+git checkout v3.3.x  # release branch
+# "v3.3.1-master" is the last release tag on the master branch, so it will
```

Comment:
Hrm... Yeah, I see how this is confusing.  Let me put the explanation in prose before the shell script instead of sticking comments in it.  I'll also use some of your wording.

Reply:
Please take a look at my latest push.  Is this clearer?